knimin/lib/data_access.py

```diff
@@ -2399,64 +2400,205 @@ def getProjectNames(self):
         sql = """SELECT project FROM project"""
         return [x[0] for x in self._con.execute_fetchall(sql)]
 
+<<<<<<< HEAD
```

Comment:
Hi @wasade , I am impressed that you reviewed the code so timely! I was still checking the code and writing comments. Sorry for the conflict. Now it is done and re-PR'ed. I am not very familiar with the GitHub review system, so I am not sure if you can see this comment. Can you please reply to confirm if you see it?